scripts/checkForClashing.js

```diff
@@ -27,6 +27,11 @@ async function main() {
 
 	for (let i = 0; i < filenames.length; i++) {
 		const filename = filenames[i];
+
+		if (filename.indexOf('.sol') != filename.length - 4) {
```

Comment:
@varasev could you specify, what does this check for?

Reply:
@vbaranov this is needed for filtering filenames by `.sol` extension in the `contracts` directory. E.g., on MacOS there can be `.DS_Store` file.

Reply:
@varasev maybe, then, it is better to change
from `if (filename.indexOf('.sol') != filename.length - 4) {`
to `if (!filename.endsWith('.sol'))`?